src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -76,8 +76,8 @@ private async Task<string> GetBuildConfig(Uri url)
     {
         var queries = HttpUtility.ParseQueryString(url.Query);
         var name = queries["name"] ?? "";
-        var repository = queries["repository_url"] ?? "";
-        var branch = queries["branch"] ?? "";
+        var repository = queries["publish_repository_url"] ?? "";
+        var branch = queries["publish_repository_url"] != queries["repository_url"] ? "main" : queries["repository_branch"] ?? "";
```

Comment:
>["repository_branch"] [](http://example.com/codeflow?start=102&length=21)

Can we handle the `repository_branch` in Docs.Build and always trust it in docfx?